### ISSUE TITLE
SimpleSecurityHandler example usage?

### ISSUE THREAD

**Issue by Bhlowe** [fpompermaier/onvif#13]
Looking at the source code, it appears that the OnvifDevice doesn't properly use the username and password for the camera. Do you have an example of wiring in the SimpleSecurityHandler?

Also, the exception thrown when the camera returns 401 Unauthorized is also really unclear-- the exception simply says "org.apache.cxf.interceptor.Fault: Could not send Message." A better exception message would be the HTTPException...

**Comment by fpompermaier** [comment]
Hi @Bhlowe, this part is taken as-is from the original code. If you want to submit a PR to fix this I'll be happy to merge it

**Comment by Bhlowe** [comment]
Somewhere in getServiceProxy you'll need to attach the SimpleSecurityHandler to the proxyFactory.

	proxyFactory.getHandlers().add(securityHandler);

I'll try to get a PR pushed-- but my code is diverging from yours a bit.. 

Wondering if you've considered adding the project to https://mvnrepository.com/ so it can be used in any java project.

**Comment by fpompermaier** [comment]
Of course this could be done. Just issue a PR and then I would register a
first public version on mvnrepository.com!

Thanks
Flavio

**Comment by Bhlowe** [comment]
Hi,
OK... I've made a bunch of changes to my fork. I'm not sure the best way to merge them with your code. But take a look and feel free to use any or suggest a PR.. https://github.com/bhlowe/onvif

Give the new discovery a try. The first should just show the URLs. The second DiscoverAndTest should attempt to log into all discovered devices. You'll need a user and password, generally. I've added a routine to use what's passed as an argument or in environment/property settings. 

This build should build with java 8.. and run on any newer java if you added the commented java 9+ dependencies. 

Maven would be nice..
